pages/fundamentals/data-types.mdx

```diff
@@ -664,6 +665,64 @@ ZonedDateTime operations:
 | ZonedDateTime - Duration      | ZonedDateTime |
 | ZonedDateTime - ZonedDateTime | Duration      |
 
+### Enum
+
+Unlike other datatypes, enums requires that they are defined first. Each named enum had a set of values it can represent.
+
+<Callout type="info">
+
+To create an Enum:
+
+```cypher
+CREATE ENUM Status VALUES { Good, Okay, Bad };
+```
+
+To see what enums exist use
+
+```cypher
+SHOW ENUMS;
+```
+
+```plaintext copy=false
+┌─────────────────────────┬─────────────────────────┐
+│ Enum Name               │ Enum Values             │
+├─────────────────────────┼─────────────────────────┤
+│ "Status"                │ ["Good", "Okay", "Bad"] │
+└─────────────────────────┴─────────────────────────┘
+```
+
+</Callout>
+
+**Literals**
+
+Inside a query you can refer to enum values in their literal form `name::value`:
+
+Examples:
+
+```cypher
+CREATE (:Machine {status: Status::Good});
+CREATE (:Machine {status: Status::Okay});
+MATCH (n:Machine) WHERE n.status = Status::Bad RETURN n;
+```
+
+**Strings**
+
+The `ToEnum()` function takes string(s) to lookup and return enum value.
+
+Examples:
+
+```cypher
+RETURN ToEnum("Status", "Good");
+RETURN ToEnum("Status::Okay");
+```
+
+<Callout type="info">
+
+Bolt protocol does not understand enums, it is marshalled to the client encoded as a map value.
+Updated clients aware of this marshalling will pretty print enum in client side results.
```

Comment:
I understand that marshalling is the correct term, but maybe "lighter" wording is needed.

Maybe this could be replaced with something like:

The Bolt protocol cannot directly handle enums. Therefore, enums are converted into a map  before being sent to the client. Clients that have been updated to recognize this conversion will convert the map back into a readable enum format and display it nicely in the client-side results.


@hal-eisen-MG @Ignition what is your take on this?

Reply:
I have no objection with less technical wording.